src/course-unit/utils.jsx

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable max-len */
+// @ts-check
+
+/**
+ * Method to return course unit view live URL path.
+ * @param {string} courseId
+ * @param {string} blockId
+ * @returns {string} {`/courses/${string}/jump_to/${string}`}
+ */
+export const getUnitViewLivePath = (courseId, blockId) => (
+  `/courses/${courseId}/jump_to/${blockId}`
+);
+
+/**
+ * Method to return course unit preview URL path.
+ * @param {string} courseId
+ * @param {string} sectionId
+ * @param {string} blockId
+ * @returns {string} {`/courses/${courseId}/courseware/interactive_demonstrations/${sectionId}/1?activate_block_id=${blockId}`}
+ */
+export const getUnitPreviewPath = (courseId, sectionId, blockId) => (
```

Comment:
@ihor-romaniuk 
I have checked, and we would need to take both links from backend
 https://github.com/raccoongang/edx-platform/pull/2484/files#diff-fcd65c5bab25478ab25e92d58b53a090ec0141ef462e24bd88052917dba52cafR73

Ticket: [AXIMST-221](https://youtrack.raccoongang.com/agiles/104-1224/105-3378?issue=AXIMST-221) Use view live and preview links from backend